gapii/cc/vulkan_mid_execution.cpp

```diff
@@ -344,6 +349,19 @@ void VulkanSpy::serializeGPUBuffers(StateSerializer* serializer) {
     }
   }
 
+  // Retrieve buffers, one by one, chunk by chunk, to prevent OOM.
+  // Prepare one staging buffer of size kChunkSizeLimit per device.
+  std::map<VkDevice, std::shared_ptr<StagingBuffer>> staging_buffers;
```

Comment:
Unless you need these to be ordered, don't use std::map.
std::unordered_map please.

Reply:
done